recipe-app/src/context/AuthContext.tsx

```diff
@@ -71,6 +71,7 @@ export function AuthProvider({ children }: Props) {
 			const userData: User = {
 				id: authData.user.uid,
 				email: authData.user.email || "",
+				role: "user",
```

Comment:
role wynieś na zewnątrz jako stałą


```suggestion
				const USER_ROLE = {
	ADMIN: "admin",
	USER: "user",
}
```